dali/test/python/test_audiodecoder_spectogram.py

```diff
@@ -0,0 +1,115 @@
+# Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
```

Comment:
You need to run this test as well.

Reply:
Why not just make it as another case in the existing test_operator_spectrogram.py  ?